org.tordini.flavio.Minitube.json

```diff
@@ -0,0 +1,83 @@
+{
+    "app-id": "org.tordini.flavio.Minitube",
+    "branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "minitube",
+    "rename-desktop-file": "minitube.desktop",
+    "rename-appdata-file": "minitube.appdata.xml",
+    "rename-icon": "minitube",
+    "copy-icon": true,	
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--talk-name=org.freedesktop.ScreenSaver",
+        "--env=QT_QPA_PLATFORM=flatpak"
+    ],
+    "modules": [
+     {
+        "name": "ffmpeg",
+        "build-options" : {
+            "arch" : {
+                "i386" : {
+                    "config-opts" : [
+                        "--arch=x86"
+                        ]
+                    },
+                "arm" : {
+                    "config-opts" : [
+                        "--arch=arm"
+                        ]
+                    }
+                }
+        },
+        "cleanup": [ "/include", "/lib/pkgconfig", "/share/ffmpeg/examples" ],
+        "config-opts": [
+            "--enable-shared",
+            "--disable-static",
+            "--enable-gnutls",
+            "--disable-doc",
+            "--disable-programs",
+            "--disable-encoders",
+            "--disable-muxers",
+            "--disable-devices"
```

Comment:
This is only for Youtube right? We don't need everything supported then only webm, vp9, opus, h264, etc.

Reply:
Yeah, only for youtube. I wasn't quite sure what they were streaming as but that makes sense.